pilot/pkg/networking/plugin/authz/rbac_v2.go

```diff
@@ -37,6 +38,13 @@ import (
 	"istio.io/istio/pilot/pkg/model"
 )
 
+const (
+	// Refer to the root namespace of the mesh.
+	// Check out root_namespace field at https://github.com/istio/api/blob/master/mesh/v1alpha1/config.proto
+	// for more details.
+	serviceRoleRootNamespaceRef = "/"
```

Comment:
s/serviceRoleRootNamespaceRef/rootNamespacePrefix
Also update the comment to make it clear this is the prefix of root namespace

Reply:
Done. Thanks!